### DISCUSSION TITLE
Getting 🚫 ope: connect ETIMEDOUT {Google IPs}

### DISCUSSION THREAD

**Discussion by ovosimpatico** [codetheweb/muse#1014] [Q&A] [answered]
![image](https://github.com/codetheweb/muse/assets/112394805/2f9d5f35-046f-4d1a-8594-daeae7705e75)

The IPs match Google ones:

![image](https://github.com/codetheweb/muse/assets/112394805/3489c64b-3fb7-4e50-9909-a4a15f8c5347)

![image](https://github.com/codetheweb/muse/assets/112394805/bfeb81a6-67bf-4412-90ba-f58b328a47d5)

Is youtube API down, or is there something else happening?

**Comment by ovosimpatico** [comment]
Even after 15 hours, I'm still getting the same issue.

**Comment by codetheweb** [comment]
Maybe YouTube is rate limiting/blocking your IP? Are you running Muse from a residential IP or on a cloud provider?

**Reply by ovosimpatico** [reply, marked as answer]
I'm running it from my own residential IP. The bot is now back to working normally as of the past 24h, I guess YouTube API was down for 2 days I guess?
